internal/docs/e2e-tests.md

```diff
@@ -1,52 +1,85 @@
 
 #### Shooted-Seed
-* Create a `shooted seed` setup by following running the script at [Gardener-Extensions-Setup](https://github.tools.sap/kubernetes/onboarding/blob/master/setup/localsetup/hacks/local-setup-extensions.sh) and deploy a local shoot in the shooted seed.(Use latest gardener version if possible)
-* After the shoot is deployed, `annotate` the `managed resource` for DWD prober present in the `garden` namespace of the seed worker plane with the following:
+
+* Create a `shooted seed` setup by following running the script
+  at [Gardener-Extensions-Setup](https://github.tools.sap/kubernetes/onboarding/blob/master/setup/localsetup/hacks/local-setup-extensions.sh)
+  and deploy a local shoot in the shooted seed.(Use latest gardener version if possible)
+* After the shoot is deployed, `annotate` the `managed resource` for DWD prober present in the `garden` namespace of the
+  seed worker plane with the following:
+
 ```bash
 # This is required to ensure that `Dependency watchdog prober pods` are not scaled up during reconciliation of the shooted seed.
 kubectl -n garden annotate managedresource dependency-watchdog-probe resources.gardener.cloud/ignore=true --overwrite
 ```
-* Check the role/clusterrole for dependency-watchdog-prober in the `garden` namespace of the seed worker plane and add a `patch` verb for deployment/scale resource if not present.
-* Check the role/rolebinding in the shoot for the dependency-watchdog-probe related service account in the `kube-system` namespace. Add rules for listing leases in the `kube-node-lease` namespace.
-* Scale down the DWD prober deployment in the garden namespace (in the shooted-seed) and start a local DWD process by providing the prober config and the kubeconfig of the shooted seed as command line flags - ```bash go run ./dwd.go prober --config-file=<path to prober config yaml> --kubeconfig=<path to shooted-seed kubeconfig yaml>```. To change the log level one can additionally pass `--zap-log-level=<loglevel>` command line flag which will be picked up zap logger at the time of initialization of DWD prober.
-* Another way of running DWD is to change the image of the DWD deployment at `imagevector/images.yaml` in the cloned gardener repo used for the setup. (This needs to be done after the script checks out the desired gardener version and before `make gardener-extensions-up` is run)
+
+* Check the role/clusterrole for dependency-watchdog-prober in the `garden` namespace of the seed worker plane and add a
+  `patch` verb for deployment/scale resource if not present.
+* Check the role/rolebinding in the shoot for the dependency-watchdog-probe related service account in the `kube-system`
+  namespace. Add rules for listing leases in the `kube-node-lease` namespace.
+* Scale down the DWD prober deployment in the garden namespace (in the shooted-seed) and start a local DWD process by
+  providing the prober config and the kubeconfig of the shooted seed as command line flags -
```

Comment:
you need to add a new line for commands that are enclosed within ```bash ``` else the formatting is not going to be correct.